app/src/main/java/com/mutualmobile/mmleave/viewmodels/AuthViewModel.kt

```diff
@@ -55,6 +65,17 @@ class AuthViewModel @Inject constructor(
         viewModelScope.launch {
             storeUserInfo.getUserAuthenticateState.collect {
                 _userAuthState.emit(it)
+                _isUserAuthenticated.value = it
+            }
+        }
+    }
+
+    private fun isAuthenticated() {
+        viewModelScope.launch {
+            // To keep Splash Icon stable for 2 Seconds
+            delay(2000)
```

Comment:
Is the application showing you quick screen changes at the start? Because if not, then we should never have to use the 'delay' method in our code. And we shouldn't keep the user waiting just to see our splashScreen. It does not matter to him.

Reply:
Okay, got it. I'll remove this delay. 
Any other issues that can be improved?

Reply:
Nope. Everything looks good for now. Keep it up 👍